lib/oxidized/nodes.rb

```diff
@@ -58,7 +58,8 @@ def show node
     def fetch node, group
       with_lock do
         i = find_node_index node
-        output = self[i].output.new
+        # Use first output type
+        output = self[i].output[0].new
```

Comment:
If we do that, then we need to check if defaults is string or array, which seems dubious practice. Also I think users will find it more confusing than comma separated.